pkg/vmimage/data/etc/systemd/system/td-agent.service.d/custom.conf

```diff
@@ -0,0 +1,4 @@
+[Unit]
```

Comment:
what's this file do? does moving to systemctl enable(s) / systemctl start(s) to near the bottom of the master and worker startup scripts mean it can be avoided?